assets/javascripts/discourse/templates/components/chat-message.hbs

```diff
@@ -33,45 +33,13 @@
     {{else}}
       <div class={{chatMessageClasses}}>
         {{#if showActions}}
-          <div class="chat-msgactions-hover">
-            <div class="chat-msgactions">
-              {{#each emojiReactions as |reaction|}}
-                {{chat-message-reaction
-                  emoji=reaction.emoji
-                  reacted=reaction.reacted
-                  react=(action "react")
-                  class="show"
-                }}
-              {{/each}}
-
-              {{#if canReact}}
-                {{d-button
-                  class="btn-flat react-btn"
-                  action=(action "startReactionForMsgActions")
-                  icon="discourse-emojis"
-                  title="chat.react"
-                }}
-              {{/if}}
-
-              {{#if canReply}}
-                {{d-button
-                  class="btn-flat reply-btn"
-                  action=(action "reply")
-                  icon="reply"
-                  title="chat.reply"
-                }}
-              {{/if}}
-
-              {{#if moreButtons.length}}
-                {{dropdown-select-box
-                  class="more-buttons"
-                  options=(hash icon="ellipsis-h")
-                  content=moreButtons
-                  onChange=(action "handleMoreButtons")
-                }}
-              {{/if}}
-            </div>
-          </div>
+          {{component (concat "chat-message-actions-" (if site.mobileView "mobile" "desktop"))
```

Comment:
It may be better to avoid `concat` here, purely to allow future devs to more easily find "chat-message-actions-desktop" and "chat-message-actions-mobile".

Reply:
initially I had duplicated both components, but I think I prefer having to deal only with updating one set of properties